features/admin.logs.v1/pages/logs-settings-page.scss

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+.log-types-grid-wrapper {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 40px;
+
+    .log-types-grid {
```

Comment:
Let's wrap this with a class named `logs-settings-page`. Expection is to scope these only to the component.

```
.logs-settings-page {
	.log-types-grid-wrapper {
	    display: flex;
	    flex-direction: row;
	    margin-bottom: 40px;
	
	    .log-types-grid {
```